src/main/java/org/jabref/preferences/JabRefPreferences.java

```diff
@@ -629,8 +629,8 @@ private JabRefPreferences() {
         defaults.put(CONFIRM_DELETE, Boolean.TRUE);
         defaults.put(DEFAULT_CITATION_KEY_PATTERN, "[auth][year]");
         defaults.put(UNWANTED_CITATION_KEY_CHARACTERS, "-`ʹ:!;?^+");
-        defaults.put(DO_NOT_RESOLVE_STRINGS_FOR, StandardField.URL.getName());
-        defaults.put(RESOLVE_STRINGS_ALL_FIELDS, Boolean.FALSE);
+        defaults.put(RESOLVE_STRINGS_FOR_FIELDS, "author;editor;editora;editorb;editorc;publisher;month;journal;journaltitle;journalsubtitle;subtitle;issuetitle;titleaddon;mainsubtitle;booktitle;shortauthor;shorteditor");
```

Comment:
I checked our help https://docs.jabref.org/advanced/strings#jabrefs-typed-strings

I think, `institution` and `publisher` should be added here, too.

Reply:
Added institution, publisher is already present